recipes/susiex/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+
+cd src
+make
+cp ../bin/SuSiEx $PREFIX/bin/SuSiEx
```

Comment:
Trivial here, but `pushd`/`popd` are a better practice for shell scripts, i.e.,

```suggestion
pushd src
make
popd
cp bin/SuSiEx $PREFIX/bin/SuSiEx
```